Updates QuaC to use with Parabricks pipeline output

ManavalanG opened the pull request:
# Pull request

**Please add a clear description of what the PR is about:**

To perform read alignment and small variant calling, we are switching from the in-house [small variant caller pipeline](https://github.com/uab-cgds-worthey/small_variant_caller_pipeline/) to the [parabricks-based pipeline](https://github.com/uab-cgds-worthey/dj-bricks). Unlike the former, the latter pipeline does not produce trimmed fastqs, and therefore QC aggregation steps concerning trimmed fastqs are removed from QuaC, as part of this PR. Note that QC jobs for fastqs are still run externally to QuaC, as before.

Changes made:

* Sample config files were updated: 
    * column `fastqc_trimmed` was removed
    * column `fastqc_raw` was renamed to `fastq`
* Following were refactored or updated to account for the above changes in the sample config file:
    * Test dataset 
    * Snakemake workflow
    * QuaC runner script
    * MultiQC config template file
    * Docs

------

**Please fill in the checklist below and comment as needed:**

- [x] Was code modified? Briefly describe. _Yes. See description above._
- [x] Was documentation modified? Briefly describe. _Yes. See description above._
- [x] Is this a bug-fix? Briefly describe. _No._
- [x] Is this a feature addition? Briefly describe. _Yes. See description above._
- [x] Did you modify QuaC-Watch config file? If so, did you modify multiqc template
  `configs/multiqc_config_template.jinja2` and script `src/quac_watch/create_mutliqc_configs.py` as necessary? _Updated `configs/multiqc_config_template.jinja2` to reflect no more fastq trimmed QC. Other two files were left untouched._
- [x] Did you perform system-level testing manually, using `----cli_cluster_config` and `--snakemake_cluster_config`
  options, as described in the [documentation](https://quac.readthedocs.io/en/stable/system_testing/)? Did it pass
  completely? If not why? _Yes, they all passed._
- [x] Updated `Changelog.md` file with change logs in recommended format?


## Anything else reviewer should know?

wilkb777 reviewed (changes requested):
Everything is looking good outside of a couple changes requested. The [example multiqc report](https://github.com/uab-cgds-worthey/quac/tree/parabricks/docs/example_output/multiqc_report.html) needs to be updated to match the removed trimmed fastq info.

ManavalanG commented:
> The [example multiqc report](https://github.com/uab-cgds-worthey/quac/tree/parabricks/docs/example_output/multiqc_report.html) needs to be updated to match the removed trimmed fastq info.

@wilkb777 Goosd catch. I made the following changes, while working to update the example multiqc report:
* Removed QC for trimmed fastq from the test datasets
* Now includes fastqc results in the multiqc general stats table. 
* Updated the example multiqc report.

wilkb777 reviewed (approved):
LGTM
